service/provider/src/main/java/gov/va/vro/service/provider/bip/service/BipClaimService.java

```diff
@@ -179,7 +179,7 @@ public FetchPdfResponse uploadPdf(FetchPdfResponse pdfResponse) {
       bipApiService.uploadEvidence(
           FileIdType.FILENUMBER,
           pdfResponse.getClaimSubmissionId(),
-          new BipFileUploadPayload(),
+          BipFileUploadPayload.builder().build(),
```

Comment:
Can you please pass the correct parameters for calling file upload here?

Reply:
Added some with notes.